### ISSUE TITLE
Molecular Events should not be entities

### ISSUE THREAD

**Issue by ukemi** [geneontology/pathways2GO#119]
In the new Reactome import, I just noticed that when there is a Molecular Event in a model it shows up as an entities in the annotation preview. This is not correct from my understanding of how molecular events are supposed to work. They are supposed to be occurents.

**Comment by ukemi** [comment]
gomodel:R-HSA-71240

**Comment by ukemi** [comment]
These appear in the annotation preview. It might be a bug in how the preview is generated or it might be a bug in how the molecular events are represented in the model?????

Example:

Molecular Event | part_of | GO:0006739 | Reactome:R-HSA-71240 | imported information used in automatic assertion |   |   | 20201124

**Comment by goodb** [comment]
weird..  tagging @balhoff and @dustine32 .

**Comment by balhoff** [comment]
My guess: these events are parts of processes in the model. That's fine. The GPAD export looks for things that have 'part of' relations to GO term instances, since 'part of' is one of the valid annotation relations. There is a filter which gets rid of things we don't want to output as annotation subjects:

https://github.com/geneontology/minerva/blob/809417340682bb982ab5770fae9c98635553667c/minerva-converter/src/main/resources/org/geneontology/minerva/legacy/sparql/gpad-basic.rq#L66-L73

The molecular event type needs to be filtered here. It's possible that some of Ben's newer work has added some kind of property to OWL classes which we can positively identify as valid annotation subjects; I'm not sure.

**Comment by ukemi** [comment]
The molecular events are definitely occurrent parts of the processes in the model. Seems like we might be able to do this in a better way than a static filter? We really need to reexamine the methods for the GPAD output.

**Comment by ukemi** [comment]
So summary take: The models are correct and this is a bug in the GPAD generator. Molecular event needs to be one of the things we filter out of the GPAD. It's weird that I hadn't noticed this before. I suspect I checked the anotations before we implemented the molecular events.

**Comment by balhoff** [comment]
> Seems like we might be able to do this in a better way than a static filter?

Yes, if all gene classes had some kind of annotation property that was available to the Minerva SPARQL query.

**Comment by goodb** [comment]
@balhoff likely related to https://github.com/geneontology/go-ontology/issues/19960 .  
Molecular Event is still being built as part of reactor.  I guess the sparql gpad code doesn't know what it is and is missing the filter becausee its asserted as a superclass of molecular function.

**Comment by ukemi** [comment]
Seems like this is now fixed. @dustine32

**Comment by dustine32** [comment]
@ukemi Correct! The observed behavior back in 2020 was Molecular Event as subject in GPAD preview annotations for pathway R-HSA-71240. In Noctua prod, the GPAD export does not currently have these "Molecular event as subject" lines. I'm suspecting the minerva GPAD export code/query was properly updated to filter these out and therefore this ticket can be closed.
